src/trade-params/TradeParams.js

```diff
@@ -101,8 +115,8 @@ export default class TradeParams extends Component {
     }
 
     componentWillUnmount() {
-        const { proposal, proposalError } = this.props;
-        if (proposalError) {
+        const { proposal, errors } = this.props;
+        if (errors.proposalError) {
```

Comment:
right now I am reluctant to change TradeParams since it's working fine, I was just trying to prevent unnecessary render

I'll go through it after refactor is mergeable